lib/app/features/chat/providers/fetch_conversation_provider.c.dart

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:ion/app/features/auth/providers/auth_provider.c.dart';
+import 'package:ion/app/features/chat/model/entities/private_direct_message_data.c.dart';
+import 'package:ion/app/features/chat/model/entities/private_message_reaction_data.c.dart';
+import 'package:ion/app/features/nostr/model/action_source.dart';
+import 'package:ion/app/features/nostr/providers/nostr_event_signer_provider.c.dart';
+import 'package:ion/app/features/nostr/providers/nostr_notifier.c.dart';
+import 'package:ion/app/services/database/ion_database.c.dart';
+import 'package:ion/app/services/nostr/ion_connect_gift_wrap_service.c.dart';
+import 'package:ion/app/services/nostr/ion_connect_seal_service.c.dart';
+import 'package:nostr_dart/nostr_dart.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'fetch_conversation_provider.c.g.dart';
+
+@riverpod
+Future<void> fetchAndSyncConversations(Ref ref) async {
+  final pubkey = ref.read(currentPubkeySelectorProvider);
```

Comment:
replace please with `ref.watch` in this place

Reply:
fixed